plaidml/edsl/tests/edsl_test-skip.intel_level_zero.txt

```diff
@@ -0,0 +1,31 @@
+CppEdsl.HigherPrecisionConstants
```

Comment:
Are all these tests still not pass?

Reply:
Some of them can pass, i have remove them from the file. Others may fail for stdx lowering, opencl spirv namespace(we just use GLSL.450), IR lowering error...

Reply:
Thanks, is this level_zero skip list the same as opencl skip list? I assume there is a little difference, but most the failed tests are due to spirv ocl ops not added in upstream yet.

Reply:
No, they shall be same,  but  some cases in opencl skip list can pass now (such like Max), so i remove them.  Not appropriate to change opencl skip list in this PR.